ui/shared/PushHealthSummary.jsx

```diff
@@ -23,36 +23,38 @@ class PushHealthSummary extends PureComponent {
 
     return (
       <div>
-        <a
-          href={getPushHealthUrl({ revision, repo: repoName })}
-          title="View Push Health details for this push"
-        >
-          <div>
-            {healthStatus !== null ? (
-              <img
-                src={needInvestigation ? broken : ok}
-                alt={needInvestigation ? 'Broken' : 'OK'}
-                width={heartSize}
-                height={heartSize}
-                className="mr-1"
-              />
-            ) : (
-              <span className="ml-1 text-darker-secondary">
-                <FontAwesomeIcon
-                  icon={faHeart}
-                  height={heartSize}
+        <span className="d-flex">
```

Comment:
This diff looks big, but I just wrapped the `<a>` tag in this span.  The rest is indentation.